src/ngraph_rewrite_pass.cc

```diff
@@ -248,6 +262,15 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
                  "Graph with Variables Rewritten for Tracking");
     }
 
+#if defined(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
```

Comment:
Similarly, this could run in the normal/grappler path without any ill-effects right? so the ifdef could be removed

Reply:
Sam as above.